packages/common-server/src/files.ts

```diff
@@ -210,3 +214,24 @@ export function removeMDExtension(nodePath: string) {
   }
   return nodePath;
 }
+
+export function readToJson(path: string) {
```

Comment:
nitpick - why not just keep this as `readJSON`?

Reply:
I don't have a good answer ;) [`8cd6916` (#3649)](https://github.com/dendronhq/dendron/pull/3649/commits/8cd6916f29f967f24d0e2b80d148bfc5948f348f). I created a tweet around this lol https://twitter.com/Namgnoi/status/1581974495107043328

Reply:
haha, nice